marketdata/stream/message.go

```diff
@@ -431,14 +434,20 @@ func (h *stocksMsgHandler) handleNews(d *msgpack.Decoder, n int) error {
 	return discardMapContents(d, n)
 }
 
+func (h *stocksMsgHandler) handleFuturesPricing(d *msgpack.Decoder, n int) error {
+	// should not happen!
+	return discardMapContents(d, n)
+}
+
 type cryptoMsgHandler struct {
-	mu                sync.RWMutex
-	tradeHandler      func(trade CryptoTrade)
-	quoteHandler      func(quote CryptoQuote)
-	barHandler        func(bar CryptoBar)
-	updatedBarHandler func(bar CryptoBar)
-	dailyBarHandler   func(bar CryptoBar)
-	orderbookHandler  func(ob CryptoOrderbook)
+	mu                    sync.RWMutex
+	tradeHandler          func(trade CryptoTrade)
+	quoteHandler          func(quote CryptoQuote)
+	barHandler            func(bar CryptoBar)
+	updatedBarHandler     func(bar CryptoBar)
+	dailyBarHandler       func(bar CryptoBar)
+	orderbookHandler      func(ob CryptoOrderbook)
+	futuresPricingHandler func(pd FuturesPricing)
```

Comment:
[minor] Why `pd`? Actually, it would be nicer if you removed all variable names from the handlers here.